clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -2394,7 +2431,12 @@ CIRGenFunction::buildAArch64BuiltinExpr(unsigned BuiltinID, const CallExpr *E,
     llvm_unreachable("NYI");
   }
   case NEON::BI__builtin_neon_vrndns_f32: {
-    llvm_unreachable("NYI");
+    mlir::Value arg0 = buildScalarExpr(E->getArg(0));
+    args.push_back(arg0);
+    return buildAArch64NeonCall(NEON::BI__builtin_neon_vrndns_f32, *this,
```

Comment:
Rename `buildAArch64NeonCall` to `buildNeonCall`. If there's anything AArch64 specific in its behavior, should be checked / asserted inside the function.